src/relu.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-import {unary} from './unary.js';
+import {unary} from './lib/unary.js';
```

Comment:
the ops in unary are WebNN ops: https://www.w3.org/TR/webnn/#api-mlgraphbuilder-unary